config/database.yml

```diff
@@ -3,8 +3,10 @@ development:
   encoding: utf8
   database: mammooc_development
   pool: 5
-  username:
+  username: postgres
```

Comment:
Instead of setting a username here, it might be a good idea to set it within the DATABASE_URL, if possible.